docs/UserGuide.md

```diff
@@ -106,7 +106,7 @@ Format: `find PREFIX/KEYWORD` where PREFIX is one of the following: `n`, `a`, `p
 **Examples:**
 * `find n/John` returns `john` and `John Doe`
 * `find a/Clementi` returns `John Doe`, `Mary Apple`<br>
```

Comment:
Could you maybe change this to `fidn a/serangoon` returns `Bernice Yu`, `David Li` instead? thanks!

Reply:
adjusted the description, closing comment